src/controller/constraints.php

```diff
@@ -184,15 +184,11 @@ protected function checkComment(string $contents, &$errors)
      */
     protected function redirectNoAdmin()
```

Comment:
```suggestion
    protected function isAdmin(string $rankUser) : bool
    {
         if ($rankUser === self::USER_RIGHT_ADMIN) {
            return true
        }
        $_SESSION['error'] = self::MESSAGE_NO_ADMIN;
        return false
    }
```

```suggestion
// usage 
if (!isAdmin($_SESSION['rankConnectedUser'])) 
{
     $this->render(....)
}
```

Reply:
OK : 

protected function isAdmin(string $rankUser): bool
  {
      if ($rankUser === self::USER_RIGHT_ADMIN) {
          return true;
      }
      $_SESSION['error'] = self::MESSAGE_NO_ADMIN;
      return false;
  }

//usage
if (!$this->isAdmin($_SESSION['rankConnectedUser'])) {
	$this->render('homepage.twig', $_SESSION);
	return;
}